### ISSUE TITLE
Create a cut down LCOW kernel config

### ISSUE THREAD

**Issue by rn** [linuxkit/linuxkit#2474]
For LCOW we currently use the standard LinuxKit kernel but omit the kernel modules from the `initrd` to minimise the size. The standard LinuxKit kernel config has a large number of features compiled in, which are not needed by LCOW so it would be nice if we had a specialised/minimal kernel config for LCOW.

This would also require a minor change to the kernel build system. We already have most of the bits in place, but we could check that if `kernel_config-${KERNEL_SERIES}-${EXTRA}` exists we just use it as it.

**Comment by arm64b** [comment]
Yes, I feel the same way. We have to refine the kernel config to remove all the un-used configs to minimize the final footprint, we absolutely need to generate the image as small as possible, that's the advantage comparing the traditional VM image (generally in GB level).
